taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -328,24 +378,37 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
                         if (!language) language = '';
                         setNewLanguageCode(ISO6391.getCode(language));
                         const isValidLanguage = ISO6391.validate(ISO6391.getCode(language))
+                        const isAlreadyShown = shownTranslationLanguages.includes(ISO6391.getCode(language))
                         const isDuplicateLanguage = renderedTranslations.some(el => (el.languageCode === ISO6391.getCode(language))) || 
                                                 nodeObject.main_language === ISO6391.getCode(language)
-                        if (isValidLanguage && !isDuplicateLanguage) {setisValidLanguageCode(true)}
-                        else {setisValidLanguageCode(false)}
+                        if (isValidLanguage  && !isDuplicateLanguage) {setLanguageAction("add")}
```

Comment:
please replace the Autocomplete with [Select](https://mui.com/material-ui/react-select/). Autocomplete are ok when you allow free text (options + something else), here we want the user to only be able to select one of any of the valid languages.